dask_cuda/benchmarks/local_cudf_shuffle.py

```diff
@@ -129,7 +129,34 @@ def bench_once(client, args, write_profile=None):
         ctx = performance_report(filename=write_profile)
 
     with ctx:
-        if args.backend in {"dask", "dask-noop"}:
+        bootstrapped = False
+        if args.backend == "rapidsmpf":
+            from rapidsmpf.config import Options
+            from rapidsmpf.examples.dask import dask_cudf_shuffle
+            from rapidsmpf.integrations.dask import bootstrap_dask_cluster
+
+            options = Options({"dask_spill_device": "0.5", "dask_statistics": "true"})
```

Comment:
Just checking, do we want these hardcoded or want to expose additional arguments to control it? It's ok if these are "safe" defaults and we're happy with these for now.